src/ServiceBus/ServiceBus/help/Set-AzServiceBusNetworkRuleSet.md

```diff
@@ -0,0 +1,256 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.ServiceBus.dll-Help.xml
+Module Name: Az.ServiceBus
+online version: https://docs.microsoft.com/en-us/powershell/module/az.servicebus/set-azservicebusnetworkruleset
+schema: 2.0.0
+---
+
+# Set-AzServiceBusNetworkRuleSet
+
+## SYNOPSIS
+Update the NetwrokruleSet of the given Namepsace in the current Azure subscription.
+
+## SYNTAX
+
+### NetworkRuleSetPropertiesSet (Default)
+```
+Set-AzServiceBusNetworkRuleSet [-ResourceGroupName] <String> [-Name] <String> [-DefaultAction <String>]
+ [-IPRule] <PSNWRuleSetIpRulesAttributes[]> [-VirtualNteworkRule] <PSNWRuleSetVirtualNetworkRulesAttributes[]>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### NetwrokruleSetInputObjectSet
+```
+Set-AzServiceBusNetworkRuleSet [-ResourceGroupName] <String> [-Name] <String>
+ [-InputObject] <PSNetworkRuleSetAttributes> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### NetworkRuleSetResourceIdParameterSet
+```
+Set-AzServiceBusNetworkRuleSet [-ResourceGroupName] <String> [-Name] <String> [-ResourceId] <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Update the NetwrokruleSet of the given Namepsace in the current Azure subscription.
+
+## EXAMPLES
+
+### Example 1 
+```powershell
+PS C:\> $IpRules = @([Microsoft.Azure.Commands.ServiceBus.Models.PSNWRuleSetIpRulesAttributes] @{IpMask = "4.4.4.4";Action = "Allow"},[Microsoft.Azure.Commands.ServiceBus.Models.PSNWRuleSetIpRulesAttributes] @{IpMask = "3.3.3.3";Action = "Allow"})
+PS C:\> $VirtualNetworkRules = @([Microsoft.Azure.Commands.ServiceBus.Models.PSNWRuleSetVirtualNetworkRulesAttributes]@{Subnet=@{Id="/subscriptions/subscriptionId/resourcegroups/ResourceGroup/providers/Microsoft.Network/virtualNetworks/sbehvnettest1/subnets/default"};IgnoreMissingVnetServiceEndpoint=$True})
+PS C:\>  Set-AzServiceBusNetworkRuleSet -ResourceGroupName v-ajnavtest -Namespace ServiceBus-Namespace1-1375 -IPRules $IpRules -VirtualNtewrokRules $VirtualNetworkRules -DefaultAction "Allow" -Debug
```

Comment:
@v-Ajnava example should be updated with fixes that were made (`-IPRules` --> `-IPRule`, typo with `-VirtualNtewrokRules`)